app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all
+  end
```

Comment:
I do not believe there is a user story that requires all the recipes on a page, therefore I don't think this `index` action is necessary.